tests/basic/pure_test.go

```diff
@@ -2365,3 +2364,279 @@ var _ = Describe("{FADAVolMigrateValidation}", func() {
+			log.InfoD("Output of multipath -ll command: %v", output)
+
+			err = LoginIntoAllControllers(n, networkInterfaces)
+			log.FailOnError(err, "Failed to login into all controllers")
+
+			// run multipath after login
+			cmd = "multipath -ll"
+			MultipathBeforeRestart, err = runCmd(cmd, n)
+			log.FailOnError(err, "Failed to run multipath -ll command on node %v", n.Name)
+			log.InfoD("Output of multipath -ll command: %v", output)
+
+		})
+
+		stepLog = "Restart portworx and check multipath consistency"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			err := Inst().V.StopDriver([]node.Node{n}, false, nil)
+			log.FailOnError(err, fmt.Sprintf("Failed to stop portworx on node [%s]", n.Name))
+			err = Inst().V.WaitDriverDownOnNode(n)
+			log.FailOnError(err, fmt.Sprintf("Driver is up on node [%s]", n.Name))
+			err = Inst().V.StartDriver(n)
+			log.FailOnError(err, fmt.Sprintf("Failed to start portworx on node [%s]", n.Name))
+			err = Inst().V.WaitDriverUpOnNode(n, addDriveUpTimeOut)
+			log.FailOnError(err, fmt.Sprintf("Driver is down on node [%s]", n.Name))
+			dash.VerifyFatal(err == nil, true,
+				fmt.Sprintf("PX is up after restarting on node [%s]", n.Name))
+
+			//run multipath after restart
+			cmd := "multipath -ll"
+			multipathAfterRestart, err := runCmd(cmd, n)
+			log.FailOnError(err, "Failed to run multipath -ll command on node %v", n.Name)
+
+			//check if the multipath entries are same before and after restart
+			dash.VerifyFatal(MultipathBeforeRestart == multipathAfterRestart, true, "Multipath entries are same before and after restart")
+
+		})
+
+		stepLog = "Delete the volume and host from the FA"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			//log out of all the controllers
+			//Run iscsiadm commands to login to the controllers
+			networkInterfaces, err := pureutils.ListAllNetworkInterfacesOnFA(FAclient)
+			log.FailOnError(err, "Failed to list all network interfaces on FA")
+
+			err = logOutOfAllControllers(n, networkInterfaces)
+			log.FailOnError(err, "Failed to logout from all controllers")
+			log.InfoD("successfully logged out of all the controllers")
+
+			//disconnect volume from host
+			_, err = pureutils.DisConnectVolumeFromHost(FAclient, hostName, volumeName)
+			log.FailOnError(err, "Failed to disconnect volume from host")
+			log.InfoD("Volume disconnected from host: %v", volumeName)
+
+			//Delete the volume
+			_, err = pureutils.DeleteVolumeOnFABackend(FAclient, volumeName)
+			log.FailOnError(err, "Failed to delete volume on FA")
+			log.InfoD("Volume deleted on FA: %v", volumeName)
+
+			//Delete the host from FAbackend
+			_, err = pureutils.DeleteHostOnFA(FAclient, hostName)
+			log.FailOnError(err, "Failed to delete host on FA")
+			log.InfoD("Host deleted on FA: %v", hostName)
+		})
+
+	})
+
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+	})
+})
+
+func LoginIntoAllControllers(n node.Node, networkInterfaces []flasharray.NetworkInterface) error {
+	var ip string
```

Comment:
Split the function to Login to one specific controller and use the function to access and login from multiple network interfaces

Reply:
Done.